french_law/json_schemas/aides_logement_schema.json

```diff
@@ -0,0 +1,2417 @@
+{
+   "type": "object",
```

Comment:
Not sure if I want to store these here, as they're more of an UI thing. I would be in favor of not including this `french_law/json_schemas` directory, and rather put those as an asset in https://github.com/CatalaLang/catala-website to be copied over from `./generate_website_assets.sh`.

Reply:
So they are directly generated by the `generate_website_assets.sh`?

Reply:
No they would still be generated by `make build_french_law_library_web_api` but not copied over to `frennch_law`, you can leave them in their `examples/*` folders where they are gitignored. Then it would be up to `generate_website_assets.sh` to pick them up in `examples/*` and copy them to the website assets folder.

Reply:
Ok, sounds great to me.